vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailViewEvents.kt

```diff
@@ -43,6 +45,11 @@ sealed class RoomDetailViewEvents : VectorViewEvents {
     data class NavigateToEvent(val eventId: String) : RoomDetailViewEvents()
     data class JoinJitsiConference(val widget: Widget, val withVideo: Boolean) : RoomDetailViewEvents()
 
+    object OpenInvitePeople : RoomDetailViewEvents()
+    object OpenSetRoomAvatarDialog : RoomDetailViewEvents()
+    object OpenRoomSettings : RoomDetailViewEvents()
```

Comment:
Is it really necessary to use the ViewModel to OpenRoomSettings? The fragment can do it directly, no?

Reply:
Yes I agree, it depends if there is some logic or not in the viewmodel before dispatching

Reply:
But it's not called from the fragment, it's called via the callback in the epoxy item.
I could eventually use the ActionSuccess to dispatch to fragment, will it make it to you?

Reply:
I wasn't aware of the onTimelineItemAction callback method. So it's ok to me.